authentication/auth.go

```diff
@@ -314,9 +314,10 @@ func getStringOrDefault(key string, values map[string][]string, defaultValue str
 }
 
 // GenerateTokenFromDefaults coverts a set of DEFAULT values into a JWT
-func GenerateTokenFromDefaults(surveyURL string, accountServiceURL string, urlValues url.Values) (token string, error string) {
+func GenerateTokenFromDefaults(surveyURL string, accountServiceURL string, accountServiceLogOutURL string, urlValues url.Values) (token string, error string) {
 	claims := make(map[string]interface{})
 	urlValues["account_service_url"] = []string{accountServiceURL}
+	urlValues["account_service_log_out_url"] = []string{accountServiceLogOutURL}
```

Comment:
Where is this coming from?
I cant see it in the docs https://ons-schema-definitions.readthedocs.io/en/latest/respondent_management_to_electronic_questionnaire.html

Reply:
Getting it from ras.  Forgot to update the document

Reply:
The doc should probably have been updated and agreed before we made this change.
As we should then be making changes to support the changed interface.